index.html

```diff
@@ -382,11 +430,11 @@
       stylesheet, an unknown resource type from another origin, and a font
       resource from another origin. Each fetch is initialized with appropriate
       request headers and priority - the unknown type is equivalent to a fetch
-      initiated `XMLHttpRequest` request. Further, these requests do not block
+      initiated <code>XMLHttpRequest</code> request. Further, these requests do not block
```

Comment:
note, you should never need to use `<code>` anywhere, if you've defined the:

```HTML
<code><dfn>XMLHttpRequest</dfn></code> interface
```
somewhere.